pwndbg/heap/ptmalloc.py

```diff
@@ -51,44 +73,51 @@ class BinType(str, Enum):
     UNSORTED = "unsortedbin"
     NOT_IN_BIN = "not_in_bin"
 
-    def valid_fields(self):
+    def valid_fields(self) -> List[str]:
         if self in [BinType.FAST, BinType.TCACHE]:
             return ["fd"]
         elif self in [BinType.SMALL, BinType.UNSORTED]:
             return ["fd", "bk"]
         elif self == BinType.LARGE:
             return ["fd", "bk", "fd_nextsize", "bk_nextsize"]
+        else:  # BinType.NOT_IN_BIN
+            return []
```

Comment:
@CptGibbon @lebr0nli are you fine with this behavior of returning an empty list for `BinType.NOT_IN_BIN` or would you prefer the old behavior of `None`?